.github/workflows/ci.yml

```diff
@@ -263,7 +263,7 @@ jobs:
             run: |
               set -e
               ./autogen.sh
-              CPPFLAGS=$(pkg-config --cflags ncurses) LDFLAGS=$(pkg-config --libs ncurses) ./configure --enable-unicode --enable-werror
+              ./configure --enable-unicode --enable-werror
```

Comment:
Looks a lot cleaner! Thanks @Explorer09

Reply:
One thing left to do is to update the netbsd/README.md to mention that configure would now favor libncurses over libcurses when the former is found. To use NetBSD's curses, you now need `--disable-unicode --with-curses=curses` two configure options.

Reply:
@Explorer09 Can you add this note for NetBSD and other platforms where it applies?

Reply:
Updated netbsd/README to mention the `--with-curses` option.